packages/queries/src/index.ts

```diff
@@ -1,7 +1,26 @@
 export { col } from './col';
+export { getTablePrimaryKeyColumns } from './schema/getTablePrimaryKeyColumns';
```

Comment:
Jim none of these functions belong here: 

These should not be part of the public API of SynthQL which we should try to keep minimal.

Am I missing something? Is there a reason why you placed them here?